web-ide/ide-server/Dockerfile

```diff
@@ -1,32 +1,43 @@
-ARG DAML_VERSION=0.12.16
+FROM openjdk:8-slim
 
-FROM "digitalasset/daml-sdk:${DAML_VERSION}-master"
+ARG DAML_VERSION=0.12.21
+USER root
 
-#strange but true...if we simply redeclare the arg defined before FROM it is visible after FROM
-ARG DAML_VERSION
+RUN addgroup --gid 1000 sdk && \
+    adduser --gecos "" --uid 1000 --gid 1000 \
+            --home /home/sdk --shell /bin/bash \
+            --disabled-password \
+            sdk
 
-RUN da list
-USER root
-# Install VS Code's deps, libxkbfile-dev and libsecret-1-dev
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - &&\
-    apt-get update && apt-get install -y \
+RUN apt-get update -y && \
+        apt-get -y install \
+        curl \
         procps \
         nodejs \
-        libxkbfile-dev \
+        # Install VS Code's deps, libxkbfile-dev and libsecret-1-dev
+        libxkbfile-dev \ 
         libsecret-1-dev &&\
+    curl -sL https://deb.nodesource.com/setup_8.x | bash - &&\
+    apt-get clean &&\
     apt-get remove -y --allow-remove-essential apt
-
+ 
 USER sdk
 ENV CODESERVER_SHA256=4fe5b4d10d3048a5e5aa3e0772c09ece56241b91e76eeaa7404b4da292442881\
-    CODESERVER_VERSION=1.939-vsc1.33.1
-RUN mkdir -p /home/sdk/.local/share/code-server/extensions/ &&\
+    CODESERVER_VERSION=1.939-vsc1.33.1\
+    PATH="${PATH}:/home/sdk/.daml/bin"
+    
+RUN cd /home/sdk/ &&\
+    curl -Lo - "https://github.com/digital-asset/daml/releases/download/v${DAML_VERSION}/daml-sdk-${DAML_VERSION}-linux.tar.gz" | tar xzvf - --strip-components 1 &&\
+    ls -l &&\
```

Comment:
Is this line necessary? Or just debug gunk that stuck around?

Reply:
oops - yea debug